lib/util/tryPackageJson.js

```diff
@@ -3,9 +3,36 @@
 const path = require('path');
 const loadJsonFile = require('load-json-file');
 
-function tryPackageJson(dir, prop) {
+function objectPath(obj, propPath) {
```

Comment:
Can you instead use ’lodash.get’ package instead of rolling out our own function?

Reply:
done

Reply:
anything else?